src/content/docs/en/guides/environment-variables.mdx

```diff
@@ -5,9 +5,44 @@ i18nReady: true
 ---
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
-:::tip[Looking for astro&#58;env\?]
-Find out more about the [experimental `astro:env` API](/en/reference/configuration-reference/#experimentalenv) for type-safe environment variables!
-:::
+Astro supports 2 main ways for handling environment variables: the `astro:env` module and Vite's built-in support.
+
+## `astro:env`
```

Comment:
Why did you not just copy the existing docs from the experimental section to start? In the worst case, just drop those in here and then we go from there!

Reply:
Well I thought it was important to say there are 2 ways to do it for people not using astro:env just yet!

Reply:
I meant, under the heading specifically for astro:env !  :smile: